<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>951</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>670</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>670</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>далее</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dichoButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод дихотомии</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goldenButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>220</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод золотого сечения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fibButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>310</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод Фибоначчи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paraButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>400</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод парабол</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brentsButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>490</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод Брента</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
